src/integrationTest/resources/application.yaml

```diff
@@ -33,8 +33,44 @@ ftp:
   reports-cron: ${FTP_REPORTS_CRON:0 30 0 * * *}
   fingerprint: SHA1:2Fo8c/96zv32xc8GZWbOGYOlRak=
   username: irrelevant
-  privateKey: ${FTP_PRIVATE_KEY}
-  publicKey: ${FTP_PUBLIC_KEY}
+  privateKey: |+
+              -----BEGIN RSA PRIVATE KEY-----
+              MIIEogIBAAKCAQEAx6JlnWeUNmS/xJZg1kcKM7bx2o/Z6/5dEzYg7MdUt2CB26nA
+              UaX36uI/5bAvCIVbYZhiImk0FPXuuvKLQ6MOygRvbwsqJwVuJ8fVZy4d2hBMxlsS
+              0/SX8TROLKUb3h+UsLJXkyPpYxRGbT9E+YWu2ISd8ba1Zz218DdQVrKFeKJnezH+
+              Lo+5kgP4IwtYAbuxi8EnLgHlae0ulhdrWAdxGxvk/fQtGNDUAG44u/DntYV/a45I
+              /30Ldyr9jhCn3E1x9318qdqui4wzcjIg8pVkAKsMVzgZNbMJS+e7+Cu5lrSkgOtO
+              eMgSa/gE0kPrxuRX7KjQ0PWG7E+YLzH/BVrhcQIDAQABAoIBAFO0ZPTt1DhGYB32
+              i53s5XsvjxL8U/xeh/ohLQb6wSuufGR1NZ1GgL/so6wUBnSFLf6q3SLeh6buQ24B
+              eqEegnn0CCFARilcXoaxhGoh7QRc50QYNbnHidnp+4k+pYOqLDc2X//jA922R51Q
+              9QPGqLxCRtNtr3wdZObW2omvisU8XSRQx0FKWlYRme4blwWLXO3XbwqQxKUmUbkD
+              ijC6TLQ7klRKZ2zLit8PUErJgKFsZKPuyIkgrfSeG4C4peUkQGaPHA6xTKqL3eW6
+              xNOjGYXCANMMLWBGUxs2N2pdD9qvsZuJ/00PZ/4TdmfRNTYKk5tvGhfaBTmiiJDj
+              q/oeSgECgYEA8uF6MVOh6z2ccW/w119g3jHhYzOSn4AT+a926Uc466VU6Npt6pzX
+              8nMJH+tJrWo1+cXXFdckgg4VWZfuDzjE7y0xI+Uo3ffbGhiPblT7sojYIXcK+UYL
+              niHkZNZIYA/6qruHRLN4pQjNFjvBDc4FJM+tG777BDkAU9bzc+RU/RECgYEA0mrq
+              Al3ou2xWm6LG1Rpc0JnTklwzYJPEXhNOGFMAy/9WeikNK5oQV6XzYUtP8xm1/+3F
+              xBmWQ+GBVa3SRCSNUCFmZf/LE7OdC+H2cfncA4ks0qAIwBY50bihv0z/wLcEPeEx
+              qrxQQ+g6EPsS7F0F5wcVYx2ygUAUaiVbYz+PHmECgYA5vz5sR/QxmD6AAhuDOZww
+              eC+chICSB8O4E+3z3Qk9jdu4WWbU52G4tX0v56Nm5d/beRyAhtyZR3vwB12EpPy9
+              2ULbw/qJfArOE5TWqMIHWt34ZvNm/S559Jj3GLOwk1wIB/pLbVh79LG+XYqzG47B
+              h9cix6+q+1qGlI7huSIz4QKBgBN0FPxxCDJDuuIas31CObCDeD8AzTMxlicIa+Dt
+              05YInkse/nu4e6hOzGiDWPnlt6DYr78Iio7uC1ILhvQL2RC+dcQaWQESSwOtOWai
+              LggZp441g3mW1HOiOToBeWesw/SgnEHntRgCIVQukjpOA1yvl/eY6Ebw0LJbj85m
+              MuohAoGAO8M64mpnVF9avuLGsv/yuX/g5/MUO/x0jZReLUmLnkpU4i5YAU2EJ8oD
+              C1ugH+oTYtMY0H4ytYPYeeEiNMyVYg3GE7U2dhpYoafRR1q6AkKr4E58Q+ThraIl
+              gH158Quekp3NyrTWuWo3OlbPwGvRvMPV0URAH9UX21mMkuqpBGE=
+              -----END RSA PRIVATE KEY-----
+  publicKey: ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDHomWdZ5Q2ZL/ElmDWRwoztvHaj9nr/l0TNiDsx1S3YIHbqcBRpffq4j/lsC8IhVthmGIiaTQU9e668otDow7KBG9vCyonBW4nx9VnLh3aEEzGWxLT9JfxNE4spRveH5SwsleTI+ljFEZtP0T5ha7YhJ3xtrVnPbXwN1BWsoV4omd7Mf4uj7mSA/gjC1gBu7GLwScuAeVp7S6WF2tYB3EbG+T99C0Y0NQAbji78Oe1hX9rjkj/fQt3Kv2OEKfcTXH3fXyp2q6LjDNyMiDylWQAqwxXOBk1swlL57v4K7mWtKSA6054yBJr+ATSQ+vG5FfsqNDQ9YbsT5gvMf8FWuFx alex@MacBook-Pro-7.local
```

Comment:
😯 . can we have _key_ specific values separately per resource file? then load them manually where needed? makes app config/properties sensible

Reply:
It would be nice to but how are these values set in production? My understanding is that this file is populated by terraform and the fields are filled in, so adding the fields to the file here mimics the production setup.

Reply:
@banderous environment variable is set